### PR TITLE
Feature/updated protect tab

### PR DESCRIPTION
**Pull request by oftheheadland**
## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3610789

## Main Changes:
* Adds the new Protect Tab content including the WSIO, PAD, and Wild and Scenic Rivers data.
* The Protected Areas Database service currently returns a CORS error when developing locally so it is not fully implemented. Once the service is fixed we can finish implementing the feature. For now the app attempts to query the service and handles the CORS error correctly.

## Steps To Test:
1. Navigate to http://localhost:3000/community/troutdale/protect
2. Check that the WSIO and Wild and Scenic River accordions are working correctly.
3. Check that the Protected Areas accordion correctly displays an error message.

### CODE REVIEW

**Comment by cschwinderg** [comment]
I started looking at this and notice a minor issue. Sometimes the photos are smaller which causes the issue in the screenshot below. This location is good for testing this http://localhost:3000/community/050500020908/protect. I think we just need to update the css or put in `<br />` tag. 

image

File: app/client/src/components/pages/Community/config.js [hunk 1]
```diff
@@ -307,8 +307,9 @@ const tabs = [
     icon: protectIcon,
     upper: protectUpper,
     lower: <Protect />,
-    layers: {}, // hide the nonprofits layer, re-add when nonprofits are approved
-    // layers: { nonprofitsLayer: true }
+    layers: {
+      wsioHealthIndexLayer: true,
+    },
```
Comment (cschwinderg) [inline app/client/src/components/pages/Community/config.js:312]:
```suggestion
    layers: {
      wsioHealthIndexLayer: true,
      wildScenicRiversLayer: false,
      protectedAreasLayer: false,
    },
```

I think this should be updated to include the other layers on this tab. This makes it so users can toggle the other protect layers on/off (via the layer list) while in full screen mode.

File: app/server/app/public/data/config/services.json [hunk 2]
```diff
@@ -45,6 +45,8 @@
     "queryStringFirstPart": "query?where=IS_DEFAULT%3D%27Y%27+and+STATE+in+%28",
     "queryStringSecondPart": "%29&outFields=*&returnGeometry=false&returnTrueCurves=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&returnDistinctValues=false&returnExtentsOnly=false&f=json"
   },
+  "wildScenicRivers": "https://services1.arcgis.com/fBc8EJBxQRMcHlei/arcgis/rest/services/WildScenicRiversV2_AGOL/FeatureServer/1/",
+  "protectedAreasDatabase": "https://gis1.usgs.gov/arcgis/rest/services/padus2/protection_mechanism_category/MapServer/0/query?f=json&where=OBJECTID=1&returnCountOnly=true",
```
Comment (cschwinderg) [inline app/server/app/public/data/config/services.json:49]: These two items need to be added to the `googleAnalyticsMapping` below. Also, should these be added to the `services-attains.json` file? I'm not sure if that one gets updated with each release or not. If it isn't updated then we can probably leave that json file as is.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
I found a couple a few small things. Other than the items below and the one above, everything looks great!

File: app/server/app/public/data/config/services.json (hunk 2 above)
Comment (oftheheadland) [reply app/server/app/public/data/config/services.json:49]: @cschwinderg good time to ask, is the services-attains.json file used for anything?

Comment (cschwinderg) [reply app/server/app/public/data/config/services.json:49]: I think it is just used for https://mywaterway-attains.app.cloud.gov/. 

@coobr01 Does the Attains version of HMW get updated when HMW is released to production?

Comment (coobr01) [reply app/server/app/public/data/config/services.json:49]: No, its a manual deployment. I can update it whenever you all want me to . But please update the attains.json file with the new services please. If you all merge the changes into ATTAINS branch, I'll manually deploy ATTAINS version.  We probably should change this if the ATTAINS instance is going to stay around I guess.

**Comment by oftheheadland** [comment]
@cschwinderg Good catch about the popup photo styling. I've updated the layers config and services config files, thank you.

**Review by cschwinderg — APPROVED** [review]
Looks good! Thanks for making those changes!

**Pull request merged**